student-store-ui/src/components/Sidebar/Sidebar.jsx

```diff
@@ -1,10 +1,37 @@
-import * as React from "react"
-import "./Sidebar.css"
+import * as React from "react";
+import "./Sidebar.css";
+
+import SidebarClosed from "./SidebarClosed/SidebarClosed";
+import SidebarActive from "./SidebarActive/SidebarActive";
+
+export default function Sidebar({
+  isActive = false,
+  handleClick,
+  cartItems,
+  products,
+  itemsOnCart,
+  setItemsOnCart,
+}) {
+  const sidebarClassName = isActive ? "sidebar open" : "sidebar closed";
+  const toggleButtonClassName = isActive
+    ? "toggle-button button open"
+    : "toggle-button button closed";
```

Comment:
This could have been written inside the className section as an alternative approach